src/chains/ethereum/ethereum/tests/api/eth/sign.test.ts

```diff
@@ -60,15 +58,12 @@ describe("api", () => {
         const msgHash = hashPersonalMessage(edgeCaseMsg);
 
         let sgn = await provider.send("eth_sign", [accounts[0], msgHex]);
-        sgn = sgn.slice(2);
 
-        const r = Buffer.from(sgn.slice(0, 64), "hex");
-        const s = Buffer.from(sgn.slice(64, 128), "hex");
-        const v = parseInt(sgn.slice(128), 16);
+        const { v, r, s } = fromRpcSig(sgn);
         const pub = ecrecover(msgHash, v, r, s);
         const addr = fromSigned(pubToAddress(pub));
-        const strAddr = "0x" + addr.toString("hex");
-        assert.deepStrictEqual(strAddr, accounts[0].toLowerCase());
+        const strAddr = Data.toString(Quantity.toBuffer(addr), 20);
```

Comment:
Same as previous comment:
```suggestion
        const strAddr = Data.toString(`0x${addr.toString(16)}`, 20);
```